docs/pt/docs/how-to/custom-request-and-route.md

```diff
@@ -0,0 +1,121 @@
+# Requisições Personalizadas e Classes da APIRoute
+
+Em algum casos, você pode querer sobreescrever a lógica usada pelas classes `Request`e `APIRoute`.
+
+Em particular, isso pode ser uma boa alternativa para uma lógica em um intermediário
```

Comment:
I don't think "Intermediário" is a good translation to middleware.

The translated docs in the tutorial section didn't translate middleware. I think it's because we don't have a good translation to it...

I would keep middleware here.